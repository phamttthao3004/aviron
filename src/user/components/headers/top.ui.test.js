import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import TopUI from './top.ui';

describe('Top UI', () => {
    it('Render Top UI component', () => {
        const props = { currentPage: undefined, clickMenu: undefined, open: undefined, setOpen: undefined, logOut: undefined };
        const com = shallow(<TopUI {...props} />);
        expect(shallowToJson(com)).toMatchSnapshot();
    });
});
