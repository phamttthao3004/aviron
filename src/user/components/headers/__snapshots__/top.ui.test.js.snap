// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Top UI Render Top UI component 1`] = `
<nav
  className="navbar navbar-inverse"
>
  <div
    className="container-fluid"
  >
    <div
      className="navbar-header"
    >
      <a
        className="navbar-brand pointer"
        href="/"
      >
        <img
          alt="Aviron logo"
          src="/images/user/logo.png"
        />
      </a>
      <div
        className="navbar-account"
      >
        <a
          className="navbar-account-avatar pointer"
          href="/"
        >
          <img
            alt="Avatar"
            src="/images/user/no-avatar.png"
          />
          <span>
            Larry Garza
          </span>
        </a>
        <span
          id="nav-account-setting"
          onClick={[Function]}
          role="presentation"
        >
          <i
            className="icon-ic_more_horiz_48px"
          />
        </span>
        <CmsPopover
          anchorId="nav-account-setting"
          className="popover-account"
          clickOutsideToClose={true}
          onClose={[Function]}
          open={false}
          placement="bottom"
        >
          <div
            style={
              Object {
                "height": "109px",
                "width": "152px",
              }
            }
          >
            <ul
              className="nav-list-item"
            >
              <li
                className=""
                onClick={[Function]}
                role="presentation"
              >
                <span>
                  Subscription
                </span>
              </li>
              <li
                className=""
                onClick={[Function]}
                role="presentation"
              >
                <span>
                  Receipt
                </span>
              </li>
              <li
                onClick={[Function]}
                role="presentation"
              >
                <span>
                  Log out
                </span>
              </li>
            </ul>
          </div>
        </CmsPopover>
      </div>
    </div>
  </div>
</nav>
`;
